templates/rankings.html.j2

```diff
@@ -63,6 +63,38 @@
                     </tr>
                 {% endfor %}
             </table>
+
+            <footer>
+                <ul>
+                    <li id="human-like">
+                       Human-like: How similar<a href="#similarity">*</a> the average response from the model
+                       is to the <a href="/human.html">human written responses</a>.                    </li>
+                    <li id="cost">Cost: The average (mean) cost of one query.</li>
+                    <li id="consistency">
+                        Consistency:
+
+Consistency is computed in this way:
+
+<ol>
+    <li>
+    Get the embedding from each of a model's responses
+    </li><li>
+    Compute the similarity<a href="#similarity">*</a> between each of those and
+    the average of those embeddings.
+    </li><li>
+    Average the result of those comparisons
+    </li>
+</ol>
+                    </li>
+                    <li id="score">Score: The score is computed by asking gpt-4-turbo to evaluate each of the model responses against a <a href="/marking.html">marking scheme</a> and summing the results.</li>
+                    <li id="similarity">Similarity: The <a href="https://en.wikipedia.org/wiki/Cosine_similarity">cosine similiarity</a> between the embeddings.
+                    </li>
+                    <li id="embedding">
+                        Embedding: The <a href="https://en.wikipedia.org/wiki/Sentence_embedding">embedding</a> is a
+                         vector which encodes the semantic content of a text.
```

Comment:
This alternative occurred to me and I thought I'd share it. Not sure if it's an improvement or not.

```suggestion
                         vector which is an encoded representation of the semantic content of a text.
```